### DISCUSSION TITLE
Support for ROS2 Jazzy?

### DISCUSSION THREAD

**Discussion by Scheggetta** [foxglove/community#1055] [General]
Is there already a timeline for the support of Jazzy for the [foxglove_bridge](https://index.ros.org/p/foxglove_bridge) package?

**Comment by defunctzombie** [comment]
There is a Jazzy package https://index.ros.org/p/foxglove_bridge/#jazzy. Could you elaborate on what else you mean by support for Jazzy?

**Reply by Scheggetta** [reply]
Where is the Jazzy tag?

![image](https://github.com/user-attachments/assets/89eab53e-c657-435f-93fe-8a09273a7cba)

**Reply by defunctzombie** [reply]
Thanks - we'll add a tag.
